tests/certification/secretstores/hashicorp/vault/metadata_helpers.go

```diff
@@ -74,7 +74,7 @@ func getComponentCapabilities(ctx flow.Context, currentGrpcPort int, targetCompo
 
 	clientCtx := context.Background()
 
-	resp, err := daprClient.GrpcClient().GetMetadata(clientCtx, &runtime.GetMetadataRequest{})
+	resp, err := daprClient.GrpcClient().GetMetadata(clientCtx, new(emptypb.Empty))
```

Comment:
This change doesn't seem right?

Reply:
this was fixing a linter issue. If I leave it as is with old val for the param, then linter is not happy.

Reply:
Not a linter issue. Per my other comment, this was due to the fact that you had downgraded the Dapr Go SDK. After you've un-downgraded that, the test now fails to build: https://github.com/dapr/components-contrib/actions/runs/9687303478/job/26731660485
Please revert this change and it should work